interface/globals.php

```diff
@@ -192,6 +192,15 @@ function undoMagicQuotes($array, $topLevel=true) {
 // Include the version file
 require_once (dirname(__FILE__) . "/../version.php");
 
+// The logging level for common/logging/logger.php
+$GLOBALS["log_level"] = "DEBUG"; // (value can be TRACE, DEBUG, INFO, WARN, or ERROR)
```

Comment:
Would be good to have an OFF, so it doesn't even try to create a log file (or else there will be a large amount of php error regarding not being able to create this file since most folks will not give the web server write privileges to the openemr directory.